api/python/tests/test_data_transfer.py

```diff
@@ -371,14 +371,25 @@ def test_upload_large_file_etag_match_metadata_mismatch(self):
 
 
     def test_multipart_upload(self):
-        path = DATA_DIR / 'large_file.npy'
+        name = 'very_large_file.bin'
+        path = pathlib.Path(name)
+
+        size = 30 * 1024 * 1024
+        chunksize = 8 * 1024 * 1024
+
+        chunks = -(-size // chunksize)
```

Comment:
Sorry, why double negative?

Reply:
Haha, this is the same as `import math`, ..., `math.ceil(size / chunksize)` - just trying to be clever and save a few bytes. I can change it to `math.ceil`.